app/models/grda_warehouse/hud/client.rb

```diff
@@ -2476,6 +2480,8 @@ def clear_view_cache
     end
 
     def most_recent_vispdat
```

Comment:
Affects this file `app/views/warehouse_reports/cohort_changes/index.xlsx.axlsx` is that ok?

Reply:
That'll be fine.  Some CoCs use the VI-SPDAT other's don't.  In this case Boston will never have any vispdats.